src/components/DataView/example/DataViewKitchenSink.tsx

```diff
@@ -656,7 +656,15 @@ function DataViewKitchenSink(): ReactElement {
 				skip: 0
 			});
 		},
-		onSavedViewRemove: function (data) {
+		onSavedViewRemove: function (data: DataViewProps["savedView"]) {
+			if(data.id === state.savedView.id) {

```

Comment:
Codacy has a fix for the issue: [Expected space(s) after "if".](https://app.codacy.com/gh/simpleviewinc/sv-mosaic/pullRequest?prid=9874252)
```suggestion
			if (data.id === state.savedView.id) {
```